README.md

```diff
@@ -31,7 +33,7 @@ If you want to use `gopherjs run` or `gopherjs test` to run the generated code l
 npm install --global source-map-support
 ```
 
-For system calls (file system access, etc.), see [this page](https://github.com/gopherjs/gopherjs/blob/master/doc/syscalls.md).
+System calls (file system access, etc.) will work only supported `GOOS` platforms (see above). See [doc/syscalls.md](https://github.com/gopherjs/gopherjs/blob/master/doc/syscalls.md) for instructions.
```

Comment:
Broken sentence.

Reply:
Thanks for spotting that, it was missing a word. I still wasn't happy with it, so I rewrote it completely in order to improve clarity. I think the new version is better.